pom.xml

```diff
@@ -150,7 +150,7 @@
                     <artifactId>maven-checkstyle-plugin</artifactId>
                     <version>3.3.1</version>
                     <configuration>
-                        <configLocation>config/maven_checks.xml</configLocation>
+                        <configLocation>google_checks.xml</configLocation>
                         <headerLocation>config/maven-header.txt</headerLocation>
                     </configuration>
                     <dependencies>
```

Comment:
question: what happens if someone breaks the rules? will the build fail? (please don't do that ;))

Reply:
no we had a lots of issue before but no fail.

Reply:
``<failsOnError> `` true would make it fail but it's default false